spec/models/manageiq/providers/kubernetes/container_manager_spec.rb

```diff
@@ -47,4 +47,40 @@
       expect(job.zone).to eq("default")
     end
   end
+
+  context "Proxy Configuration" do
```

Comment:
@chessbyte @Fryguy is there anyone that could review this test and guide us here? I am sure this is not by the book (especially fiddling with `WebMock.allow_net_connect`). Thanks.